public/403.html

```diff
@@ -0,0 +1,22 @@
+
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  </head>
+
+  <body>
+    <div class="container">
```

Comment:
Please add the noscript bit here for Google Tag Manager (GTM for all pages)


<!-- Google Tag Manager (noscript) -->
--
  | <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-N9PZZHW"
  | height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
  | <!-- End Google Tag Manager (noscript) -->